requirements-cuda.txt

```diff
@@ -0,0 +1,6 @@
+ray >= 2.9
+torch == 2.1.2
+xformers == 0.0.23.post1  # Required for CUDA 12.1.
+pynvml == 11.5.0
+triton >= 2.1.0
+outlines == 0.0.34  # Requires torch >= 2.1.0
```

Comment:
I put `outlines` to `requirements-cuda` because it depends on `torch>=2.1.0` (`pip install outlines` installs the CUDA version of PyTorch).

Reply:
Moved it back to `requirements-common.txt`.